### DISCUSSION TITLE
[Released] Team API tokens

### DISCUSSION THREAD

**Discussion by baked-dev** [vercel/community#354] [Idea]
I'm planning to run an api that integrates the vercel api and would prefer to not use personal tokens for that usecase. Is there a way to do this already and if not I would like to suggest that feature.

**Comment by leerob** [comment]
https://vercel.com/changelog/access-tokens-can-now-be-scoped-to-teams

**Reply by frdwhite24** [reply]
@leerob i'm sorry to bring up this discussion again, but the above documentation doesn't solve the original feature request. Or at least not all use cases of it. Currently all deploys are under my name in my work's CI pipeline now that we're using Vercel CLI to deploy our apps. The token is scoped correctly to the team, but is obviously still associated with my user. Were I to leave, they'd have to rotate out these tokens. Is there any documentation on this case and how to associate the token with the team instead?

**Reply by jonworek** [reply]
I couldn't find any documentation to this effect, as I had exactly the same concern. The only "workaround" there seems to be is to use a deploy hook instead of making calls through the vercel API with a user-issued token. It doesn't allow quite the same capability but it might serve your needs depending on your use case.
